### PR TITLE
DAOS-5213 tools: Add bash completion for control plane tools

### PR DESCRIPTION
**Pull request by mjmac**
Allows for tab-completion of subcommands and flags. A subsequent
commit may add completion support for the daos user tool.

### CODE REVIEW

**Comment by mjmac** [comment]
> 
> 
> Seems fine. Just the one question.

I didn't see a question, did you forget to submit it?

File: utils/rpms/daos.spec [hunk 1]
```diff
@@ -217,6 +217,7 @@ mkdir -p %{?buildroot}/%{_unitdir}
 install -m 644 utils/systemd/%{server_svc_name} %{?buildroot}/%{_unitdir}
 install -m 644 utils/systemd/%{agent_svc_name} %{?buildroot}/%{_unitdir}
 mkdir -p %{?buildroot}/%{conf_dir}/certs/clients
+mv %{?buildroot}/%{_prefix}/etc/bash_completion.d %{?buildroot}/%{_sysconfdir}
```
Comment (brianjmurrell) [inline utils/rpms/daos.spec:220]: How come this isn't covered by the patch you did a while ago to move `/usr/etc/` to `/etc/`?

**Comment by brianjmurrell** [comment]
> I didn't see a question, did you forget to submit it?

Strange.  I don't know how one can provide a review response without also submitting the comments for the review.  Anyway, question added.

File: utils/rpms/daos.spec (hunk 1 above)
Comment (mjmac) [reply utils/rpms/daos.spec:220]:
```%define conf_dir %{_sysconfdir}/daos

scons %{?_smp_mflags}      \
      --config=force       \
      --no-rpath           \
      USE_INSTALLED=all    \
      CONF_DIR=%{conf_dir} \
      PREFIX=%{?buildroot}
```

Seemed like overkill to add a new scons variable just for this...

Comment (brianjmurrell) [reply utils/rpms/daos.spec:220]: Oh yes.  I recall now.  We are telling `scons` to use `/etc/daos`, not `daos/' in `/etc/`.

**Comment by mjmac** [comment]
Known failures:
  * [DAOS-5493](https://jira.hpdd.intel.com/browse/DAOS-5493)
  * [DAOS-5435](https://jira.hpdd.intel.com/browse/DAOS-5435)

**Pull request merged**